distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -3506,6 +3519,8 @@ Once everything is configured and running, hooking up an existing {branding} ins
 
 The {branding} should now use the SSO/STS/LDAP servers when it attempts to authenticate a user upon an attempted log in.
 
+//  DDF-1796: Hardening Section Needs a lot of Clean up
```

Comment:
It's going to be updated to provide minimal information that applies to any environment where DDF may be deployed to. If we feel that everything can be covered in the other existing security related sections, we will remove the section. See https://codice.atlassian.net/browse/DDF-1796.